doc/quick-start-user-guide.md

```diff
@@ -68,15 +60,64 @@ clientaddress = "192.168.56.101:24007"
 etcdcurls = "http://192.168.56.101:2379"
 etcdpurls = "http://192.168.56.101:2380"
 ```
-
 Replace the IP address accordingly on each node.
 
-**Start glusterd2 process:** Glusterd2 is not a daemon and currently can run only in the foreground.
+### Using external etcd instead of embedded etcd.
+
+By default glusterd2 uses embedded etcd, but glusterd2 also supports usage of external etcd.
+
+To provide external etcd details. Edit glusterd2 config file by adding `noembed` option and specifying
+the etcd endpoint:
+
+```toml
+etcdendpoints = "http://[ip_address]:[port]"
+noembed = true
+```
+Provide ip-address and port of node where etcd is running.
+
+### Running glusterd2 for RPM installation
+
+**Enable glusterd2 service:** Enable glusterd2 service to ensure that glusterd2 service starts automatically when system starts.
+To enable glusterd2 service run:
+
+```sh
+# systemctl enable glusterd2
+```
+
+**Start glusterd2 service:** To start glusterd2 process run:
+
+```sh
+# systemctl start glusterd2
+```
+
+Check the status of glusterd2 service:
+
+```sh
+# systemctl status glusterd2
+```
+Please ensure that glusterd2 service status is "active (runnning)" before proceeding.
+
+
+### Running glusterd2 for binaries installation
+
+**Start glusterd2 process:** Create a config file and provide the path of config file while running glusterd2 as shown below.
 
 ```sh
 # ./glusterd2 --config conf.toml
 ```
 
+### Authentication
+
+In glusterd2, REST API authentication is enabled by default. To disable rest authentication add `restauth=false` in Glusterd2 config file(`/etc/glusterd2/glusterd2.toml`) or the custom config file provided by you (conf.toml as per above example)
```

Comment:
Is this something that is recommended? If not, perhaps it is prudent to add a disclaimer like "the developers would recommend that the ReST API authentication is not disabled"

Reply:
IMO sure will add the disclaimer, but IMO for the pupose of speeding up the setup, its better to let user disable the authentication for trying out GD2.